public/pages/CreateTrigger/components/TriggerExpressions/TriggerExpressions.js

```diff
@@ -59,38 +58,42 @@ class TriggerExpressions extends Component {
   renderPopover() {
     return (
       <div style={POPOVER_STYLE}>
-        <EuiExpression>
-          <EuiFlexGroup style={{ maxWidth: 600 }}>
-            <EuiFlexItem grow={false} style={{ width: 150 }}>
-              <Field
-                name="thresholdEnum"
-                render={({ field: { onBlur, ...rest }, form: { touched, errors } }) => (
-                  <EuiFormRow
-                    isInvalid={touched.thresholdEnum && !!errors.thresholdEnum}
-                    error={errors.thresholdEnum}
-                  >
-                    <EuiSelect options={THRESHOLD_ENUM_OPTIONS} {...rest} />
-                  </EuiFormRow>
-                )}
-              />
-            </EuiFlexItem>
+        <EuiFlexGroup
+          style={{
+            maxWidth: 600,
+            padding: '20px',
+            whiteSpace: 'nowrap',
```

Comment:
Looks like you can use EXPRESSION_STYLE here

Reply:
Looks like we have expressions in different pages, can consolidate later so we don't have duplicated styles.